addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -4365,7 +4365,7 @@ var SnippetsMenu = Widget.extend({
 
             // Always enable the deepest editor whose DOM target is still inside
             // the document.
-            if (editors[0] !== this._enabledEditorHierarchy[0]) {
+            if (editors.length && editors[0] !== this._enabledEditorHierarchy[0]) {
```

Comment:
Why this change?

Reply:
This change is for issue 2 of traceback when continuously adding and removing slide from sidebar.